documentation/modules/cruise-control/proc-deploying-cruise-control.adoc

```diff
@@ -0,0 +1,91 @@
+// This module is included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='proc-deploying-cruise-control-{context}']
+= Deploying Cruise Control
+
+Deploy Cruise Control through the `Kafka` resource in the `cruiseControl` properties.
+
+Refer to the xref:ref-sample-kafka-resource-config-deployment-configuration-kafka[sample Kafka YAML configuration] for an overview of the `Kafka` resource and its properties.
+
+The properties relevant to the Cruise Control configuration are shown in this procedure.
+
+You can configure these properties as part of a deployment or redeployment of the Kafka cluster.
+
+.Prerequisites
+
+* A Kubernetes cluster
+* A running Cluster Operator
+
+.Procedure
+
+. Edit the `cruiseControl` properties for the `Kafka` resource.
++
+The properties you can configure are shown in this example configuration:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  # ...
+  cruiseControl:
+    replicas: 1 <1>
+    config: <2>
+      default.goals: >
+         com.linkedin.kafka.cruisecontrol.analyzer.goals.RackAwareGoal,
```

Comment:
If this is a list then each line should start with a "-" and there should not be "," at the end of each line

Reply:
Not meant to be a yaml list, just a multiline yaml string, (too make it easier to read). It originally looked like this: 
```
cruiseControl:    
  config:
    default.goals: com.linkedin.kafka.cruisecontrol.analyzer.goals.RackAwareGoal,com.linkedin.kafka.cruisecontrol.analyzer.goals.ReplicaCapacityGoal
    cpu.balance.threshold: 1.1
    ...
```

Reply:
Ok but this should probably be a YAML list to make it easier to read and write for a user. Would this require operator side parsing?

Reply:
It would require some additional parsing logic in the operator, but it can be done.  I can update it in the docs here after I implement the parsing logic